docs/components/ColorTile.js

```diff
@@ -5,34 +5,42 @@ import { Box, Text } from 'gestalt';
 import tokens from 'gestalt-design-tokens/dist/json/variables.json';
 
 type Props = {|
-  name: string,
-  tokenId: string,
-  number: number,
-  textColor: 'darkGray' | 'white',
+  fullTokenName: string,
```

Comment:
What is the difference between `fullTokenName` and `name`? Should `name` be updated to `displayName`? Should `fullTokenName` just be `tokenName`?

Reply:
Simplified the API for this cmp, should be easier now